### ISSUE TITLE
conda 4.12 still parses python3.10 as 3.1

### ISSUE THREAD

**Issue by ghylander** [conda/conda#11353]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help
- [X] I looked at the [docs](https://docs.conda.io/projects/conda/en/latest/user-guide/index.html) and didn't see anything to help

### What happened?

conda did not parse the full name of python 3.10, so I googled it and found some issue in the repo. Replies by conda team reported that it would be fixed in version 4.11.1 (and so does the release description), however, even after updating to v4.12, I still get the same issue.

I tried to restart the WSL machine but issue persist

### Conda info

```shell
active environment : CH
    active env location : /home/ghylander/miniconda3/envs/CH
            shell level : 2
       user config file : /home/ghylander/.condarc
 populated config files :
          conda version : 4.12.0
    conda-build version : not installed
         python version : 3.9.5.final.0
       virtual packages : __linux=5.10.60.1=0
                          __glibc=2.27=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /home/ghylander/miniconda3  (writable)
      conda av data dir : /home/ghylander/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/ghylander/miniconda3/pkgs
                          /home/ghylander/.conda/pkgs
       envs directories : /home/ghylander/miniconda3/envs
                          /home/ghylander/.conda/envs
               platform : linux-64
             user-agent : conda/4.12.0 requests/2.27.1 CPython/3.9.5 Linux/5.10.60.1-microsoft-standard-WSL2 ubuntu/18.04.5 glibc/2.27
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```

### Conda config

```shell
nothing is returned by the command
```

### Conda list

```shell
# packages in environment at /home/ghylander/miniconda3/envs/CH:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       1_gnu    conda-forge
absl-py                   1.0.0                    pypi_0    pypi
alembic                   1.7.5                    pypi_0    pypi
anyio                     3.5.0           py310hff52083_0    conda-forge
apache-beam               2.36.0                   pypi_0    pypi
argon2-cffi               21.3.0                   pypi_0    pypi
argon2-cffi-bindings      21.2.0          py310h6acc77f_1    conda-forge
asttokens                 2.0.5                    pypi_0    pypi
astunparse                1.6.3                    pypi_0    pypi
attrs                     21.4.0                   pypi_0    pypi
autopage                  0.5.0                    pypi_0    pypi
avro-python3              1.10.2                   pypi_0    pypi
babel                     2.9.1                    pypi_0    pypi
backcall                  0.2.0                    pypi_0    pypi
backports                 1.0                        py_2    conda-forge
backports-functools-lru-cache 1.6.4                    pypi_0    pypi
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
bcrypt                    3.2.0                    pypi_0    pypi
black                     21.12b0                  pypi_0    pypi
bleach                    4.1.0                    pypi_0    pypi
bottle                    0.12.18                  pypi_0    pypi
brotli                    1.0.9                h7f98852_6    conda-forge
brotli-bin                1.0.9                h7f98852_6    conda-forge
brotlipy                  0.7.0           py310h6acc77f_1003    conda-forge
bzip2                     1.0.8                h7f98852_4    conda-forge
ca-certificates           2022.2.1             h06a4308_0    defaults
cachetools                5.0.0                    pypi_0    pypi
certifi                   2021.10.8       py310hff52083_1    conda-forge
cffi                      1.15.0          py310hd667e15_1    defaults
charset-normalizer        2.0.10                   pypi_0    pypi
click                     8.0.3           py310hff52083_1    conda-forge
cliff                     3.10.0                   pypi_0    pypi
cloudpickle               2.0.0                    pypi_0    pypi
cmaes                     0.8.2                    pypi_0    pypi
cmd2                      2.3.3           py310hff52083_1    conda-forge
colorama                  0.4.4                    pypi_0    pypi
colorlog                  6.6.0           py310hff52083_0    conda-forge
commonmark                0.9.1                    pypi_0    pypi
contextlib2               21.6.0                   pypi_0    pypi
control                   0.9.1              pyhd8ed1ab_2    conda-forge
coverage                  6.3.2                    pypi_0    pypi
crcmod                    1.7                      pypi_0    pypi
cryptography              36.0.0          py310h9ce1e76_0    defaults
cycler                    0.11.0                   pypi_0    pypi
cython                    0.29.28                  pypi_0    pypi
dataclasses               0.8                      pypi_0    pypi
dbus                      1.13.18              hb2f20db_0    anaconda
debugpy                   1.5.1           py310h122e73d_0    conda-forge
decorator                 5.1.1                    pypi_0    pypi
defusedxml                0.7.1                    pypi_0    pypi
dill                      0.3.1.1                  pypi_0    pypi
dm-tree                   0.1.6                    pypi_0    pypi
docopt                    0.6.2                    pypi_0    pypi
entrypoints               0.3                      pypi_0    pypi
et-xmlfile                1.1.0                    pypi_0    pypi
executing                 0.8.2                    pypi_0    pypi
expat                     2.2.10               he6710b0_2    anaconda
fastavro                  1.4.9                    pypi_0    pypi
flatbuffers               1.12                     pypi_0    pypi
flit-core                 3.6.0                    pypi_0    pypi
fontconfig                2.13.0               h9420a91_0    anaconda
fonttools                 4.29.0          py310h6acc77f_0    conda-forge
freetype                  2.10.4               h0708190_1    conda-forge
gast                      0.4.0                    pypi_0    pypi
gin-config                0.5.0                    pypi_0    pypi
glib                      2.56.2               hd408876_0    anaconda
google-api-core           2.5.0                    pypi_0    pypi
google-api-python-client  2.37.0                   pypi_0    pypi
google-auth               2.6.0                    pypi_0    pypi
google-auth-httplib2      0.1.0                    pypi_0    pypi
google-auth-oauthlib      0.4.6                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
googleapis-common-protos  1.54.0                   pypi_0    pypi
greenlet                  1.1.2           py310h122e73d_1    conda-forge
grpcio                    1.43.0                   pypi_0    pypi
gst-plugins-base          1.14.0               hbbd80ab_1    anaconda
gstreamer                 1.14.0               hb453b48_1    anaconda
h5py                      3.6.0                    pypi_0    pypi
hdfs                      2.6.0                    pypi_0    pypi
httplib2                  0.19.1                   pypi_0    pypi
icu                       58.2                 he6710b0_3    anaconda
idna                      3.3                      pypi_0    pypi
imgviz                    1.4.1                    pypi_0    pypi
importlib-metadata        4.10.1          py310hff52083_0    conda-forge
importlib-resources       5.4.0                    pypi_0    pypi
importlib_resources       5.4.0              pyhd8ed1ab_0    conda-forge
iniconfig                 1.1.1                    pypi_0    pypi
ipykernel                 6.9.1           py310hfdc917e_0    conda-forge
ipython                   8.0.1           py310hff52083_0    conda-forge
ipython-genutils          0.2.0                    pypi_0    pypi
ipython_genutils          0.2.0                      py_1    conda-forge
jbig                      2.1               h7f98852_2003    conda-forge
jedi                      0.18.1          py310hff52083_0    conda-forge
jinja2                    3.0.3                    pypi_0    pypi
joblib                    1.1.0                    pypi_0    pypi
jpeg                      9e                   h7f98852_0    conda-forge
json5                     0.9.5                    pypi_0    pypi
jsonschema                4.4.0                    pypi_0    pypi
jupyter_client            7.1.2              pyhd8ed1ab_0    conda-forge
jupyter_core              4.9.2           py310hff52083_0    conda-forge
jupyter_server            1.13.5             pyhd8ed1ab_1    conda-forge
jupyterlab                3.2.9              pyhd8ed1ab_0    conda-forge
jupyterlab_pygments       0.1.2              pyh9f0ad1d_0    conda-forge
jupyterlab_server         2.10.3             pyhd8ed1ab_0    conda-forge
kaggle                    1.5.12                   pypi_0    pypi
keras                     2.8.0                    pypi_0    pypi
keras-nightly             2.9.0.dev2022020208          pypi_0    pypi
keras-preprocessing       1.1.2                    pypi_0    pypi
kernel-driver             0.0.7                    pypi_0    pypi
kiwisolver                1.3.2           py310h91b1402_1    conda-forge
labelme                   4.6.0                    pypi_0    pypi
lcms2                     2.12                 hddcbb42_0    conda-forge
ld_impl_linux-64          2.36.1               hea4e1c9_2    conda-forge
lerc                      3.0                  h9c3ff4c_0    conda-forge
libblas                   3.9.0           13_linux64_openblas    conda-forge
libbrotlicommon           1.0.9                h7f98852_6    conda-forge
libbrotlidec              1.0.9                h7f98852_6    conda-forge
libbrotlienc              1.0.9                h7f98852_6    conda-forge
libcblas                  3.9.0           13_linux64_openblas    conda-forge
libclang                  13.0.0                   pypi_0    pypi
libdeflate                1.8                  h7f98852_0    conda-forge
libffi                    3.3                  he6710b0_2    anaconda
libgcc-ng                 11.2.0              h1d223b6_12    conda-forge
libgfortran-ng            11.2.0              h69a702a_12    conda-forge
libgfortran5              11.2.0              h5c6108e_12    conda-forge
libgomp                   11.2.0              h1d223b6_12    conda-forge
liblapack                 3.9.0           13_linux64_openblas    conda-forge
libnsl                    2.0.0                h7f98852_0    conda-forge
libopenblas               0.3.18          pthreads_h8fe5266_0    conda-forge
libpng                    1.6.37               h21135ba_2    conda-forge
libprotobuf               3.19.3               h780b84a_0    conda-forge
libsodium                 1.0.18               h36c2ea0_1    conda-forge
libstdcxx-ng              11.2.0              he4da1e4_12    conda-forge
libtiff                   4.3.0                h6f004c6_2    conda-forge
libuuid                   1.0.3                h1bed415_2    anaconda
libwebp-base              1.2.2                h7f98852_1    conda-forge
libxcb                    1.14                 h7b6447c_0    anaconda
libxml2                   2.9.10               hb55368b_3    anaconda
libzlib                   1.2.11            h36c2ea0_1013    conda-forge
lvis                      0.5.3                    pypi_0    pypi
lxml                      4.8.0                    pypi_0    pypi
lz4-c                     1.9.3                h9c3ff4c_1    conda-forge
mako                      1.1.6                    pypi_0    pypi
markdown                  3.3.6                    pypi_0    pypi
markupsafe                2.0.1           py310h6acc77f_1    conda-forge
matplotlib                3.2.2                    pypi_0    pypi
matplotlib-inline         0.1.3                    pypi_0    pypi
mistune                   0.8.4           py310h6acc77f_1005    conda-forge
mock                      4.0.3                    pypi_0    pypi
munkres                   1.1.4                    pypi_0    pypi
mypy_extensions           0.4.3           py310hff52083_4    conda-forge
nbclassic                 0.3.5              pyhd8ed1ab_0    conda-forge
nbclient                  0.5.11             pyhd8ed1ab_0    conda-forge
nbconvert                 6.4.2           py310hff52083_0    conda-forge
nbformat                  5.1.3              pyhd8ed1ab_0    conda-forge
nbterm                    0.0.12                   pypi_0    pypi
ncurses                   6.2                  h58526e2_4    conda-forge
nest-asyncio              1.5.4                    pypi_0    pypi
notebook                  6.4.8              pyha770c72_0    conda-forge
numexpr                   2.8.1                    pypi_0    pypi
numpy                     1.21.5                   pypi_0    pypi
oauth2client              4.1.3                    pypi_0    pypi
oauthlib                  3.2.0                    pypi_0    pypi
object-detection          0.1                      pypi_0    pypi
olefile                   0.46                     pypi_0    pypi
onnx                      1.10.2          py310hf6c9c61_2    conda-forge
opencv-python             4.5.5.62                 pypi_0    pypi
opencv-python-headless    4.5.5.62                 pypi_0    pypi
openjpeg                  2.4.0                hb52868f_1    conda-forge
openpyxl                  3.0.9                    pypi_0    pypi
openssl                   1.1.1n               h7f8727e_0    defaults
opt-einsum                3.3.0                    pypi_0    pypi
optuna                    2.10.0                   pypi_0    pypi
optuna-dashboard          0.5.0                    pypi_0    pypi
orjson                    3.6.7                    pypi_0    pypi
packaging                 21.3                     pypi_0    pypi
pandas                    1.4.0           py310hb5077e9_0    conda-forge
pandoc                    2.17.0.1             h7f98852_0    conda-forge
pandocfilters             1.5.0                    pypi_0    pypi
paramiko                  2.9.2                    pypi_0    pypi
parso                     0.8.3                    pypi_0    pypi
pathspec                  0.9.0                    pypi_0    pypi
patsy                     0.5.2                    pypi_0    pypi
pbr                       5.8.0                    pypi_0    pypi
pcre                      8.44                 he6710b0_0    anaconda
pexpect                   4.8.0                    pypi_0    pypi
pickleshare               0.7.5                    pypi_0    pypi
pillow                    8.4.0           py310h07f4688_0    conda-forge
pip                       22.0.3                   pypi_0    pypi
platformdirs              2.3.0                    pypi_0    pypi
pluggy                    1.0.0                    pypi_0    pypi
portalocker               2.4.0                    pypi_0    pypi
prettytable               3.0.0                    pypi_0    pypi
prometheus-client         0.13.0                   pypi_0    pypi
prometheus_client         0.13.0             pyhd8ed1ab_0    conda-forge
promise                   2.3                      pypi_0    pypi
prompt-toolkit            3.0.24                   pypi_0    pypi
proto-plus                1.20.3                   pypi_0    pypi
protobuf                  3.19.1                   pypi_0    pypi
psutil                    5.9.0                    pypi_0    pypi
ptyprocess                0.7.0                    pypi_0    pypi
pure-eval                 0.2.2                    pypi_0    pypi
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
py                        1.11.0                   pypi_0    pypi
py-cpuinfo                8.0.0                    pypi_0    pypi
pyarrow                   6.0.1                    pypi_0    pypi
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pycocotools               2.0                      pypi_0    pypi
pycparser                 2.21                     pypi_0    pypi
pydot                     1.4.2                    pypi_0    pypi
pygments                  2.11.2                   pypi_0    pypi
pymongo                   3.12.3                   pypi_0    pypi
pynacl                    1.5.0                    pypi_0    pypi
pyopenssl                 21.0.0                   pypi_0    pypi
pyparsing                 2.4.7                    pypi_0    pypi
pyperclip                 1.8.2                    pypi_0    pypi
pyqt                      5.9.2           py310h295c915_6    defaults
pyqt5                     5.15.6                   pypi_0    pypi
pyqt5-qt5                 5.15.2                   pypi_0    pypi
pyqt5-sip                 12.9.1                   pypi_0    pypi
pyrsistent                0.18.1          py310h6acc77f_0    conda-forge
pysocks                   1.7.1           py310hff52083_4    conda-forge
pytest                    7.0.1                    pypi_0    pypi
pytest-cov                3.0.0                    pypi_0    pypi
python                    3.10.0               h12debd9_1    defaults
python-dateutil           2.8.2                    pypi_0    pypi
python-slugify            6.0.1                    pypi_0    pypi
python_abi                3.10                    2_cp310    conda-forge
pytz                      2021.3                   pypi_0    pypi
pywin32                   303             py310hff52083_0    conda-forge
pyyaml                    5.4.1                    pypi_0    pypi
pyzmq                     22.3.0          py310h675a958_1    conda-forge
qt                        5.9.7                h5867ecd_1    anaconda
qtpy                      2.0.1                    pypi_0    pypi
readline                  8.1                  h46c0cb4_0    conda-forge
regex                     2022.1.18                pypi_0    pypi
requests                  2.27.1                   pypi_0    pypi
requests-oauthlib         1.3.1                    pypi_0    pypi
rich                      11.2.0                   pypi_0    pypi
rsa                       4.8                      pypi_0    pypi
sacrebleu                 2.0.0                    pypi_0    pypi
scikit-learn              1.0.2           py310h1246948_0    conda-forge
scipy                     1.7.3           py310hea5193d_0    conda-forge
seaborn                   0.11.2                   pypi_0    pypi
seaborn-base              0.11.2             pyhd8ed1ab_0    conda-forge
send2trash                1.8.0                    pypi_0    pypi
sentencepiece             0.1.96                   pypi_0    pypi
seqeval                   1.2.2                    pypi_0    pypi
setuptools                60.5.0          py310hff52083_0    conda-forge
sip                       4.19.13         py310h295c915_0    defaults
six                       1.16.0                   pypi_0    pypi
slycot                    0.4.0.0         py310h5cd9050_8    conda-forge
sniffio                   1.2.0           py310hff52083_2    conda-forge
sqlalchemy                1.4.31          py310h6acc77f_0    conda-forge
sqlite                    3.37.0               h9cd32fc_0    conda-forge
sshtunnel                 0.4.0                    pypi_0    pypi
stack-data                0.1.4                    pypi_0    pypi
stack_data                0.1.4              pyhd8ed1ab_0    conda-forge
statsmodels               0.13.1          py310h96516ba_0    conda-forge
stevedore                 3.5.0           py310hff52083_2    conda-forge
tables                    3.7.0                    pypi_0    pypi
tabulate                  0.8.9                    pypi_0    pypi
tb-nightly                2.9.0a20220202           pypi_0    pypi
tensorboard               2.8.0                    pypi_0    pypi
tensorboard-data-server   0.6.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.1                    pypi_0    pypi
tensorflow                2.8.0                    pypi_0    pypi
tensorflow-addons         0.16.1                   pypi_0    pypi
tensorflow-datasets       4.5.2                    pypi_0    pypi
tensorflow-hub            0.12.0                   pypi_0    pypi
tensorflow-io             0.24.0                   pypi_0    pypi
tensorflow-io-gcs-filesystem 0.24.0                   pypi_0    pypi
tensorflow-metadata       1.6.0                    pypi_0    pypi
tensorflow-model-optimization 0.7.1                    pypi_0    pypi
tensorflow-text           2.8.1                    pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
terminado                 0.12.1          py310hff52083_1    conda-forge
testpath                  0.5.0                    pypi_0    pypi
text-unidecode            1.3                      pypi_0    pypi
tf-estimator-nightly      2.9.0.dev2022022009          pypi_0    pypi
tf-models-official        2.8.0                    pypi_0    pypi
tf-nightly                2.9.0.dev20220220          pypi_0    pypi
tf-slim                   1.1.0                    pypi_0    pypi
tf2onnx                   1.10.0                   pypi_0    pypi
threadpoolctl             3.0.0                    pypi_0    pypi
tk                        8.6.11               h27826a3_1    conda-forge
tomli                     1.2.2                    pypi_0    pypi
tornado                   6.1             py310h6acc77f_2    conda-forge
tqdm                      4.62.3                   pypi_0    pypi
traitlets                 5.1.1                    pypi_0    pypi
typed-ast                 1.5.2           py310h6acc77f_0    conda-forge
typeguard                 2.13.3                   pypi_0    pypi
typer                     0.4.0                    pypi_0    pypi
typing-extensions         4.0.1                    pypi_0    pypi
typing_extensions         4.0.1              pyha770c72_0    conda-forge
tzdata                    2021e                he74cb21_0    conda-forge
unicodedata2              14.0.0          py310h6acc77f_0    conda-forge
uritemplate               4.1.1                    pypi_0    pypi
urllib3                   1.26.8                   pypi_0    pypi
wcwidth                   0.2.5                    pypi_0    pypi
webencodings              0.5.1                    pypi_0    pypi
websocket-client          1.2.3                    pypi_0    pypi
werkzeug                  2.0.2                    pypi_0    pypi
wetb                      0.1.0                    pypi_0    pypi
wheel                     0.37.1                   pypi_0    pypi
wrapt                     1.13.3                   pypi_0    pypi
xlwt                      1.3.0                    pypi_0    pypi
xz                        5.2.5                h516909a_1    conda-forge
yaml                      0.2.5                h7f98852_2    conda-forge
zeromq                    4.3.4                h9c3ff4c_1    conda-forge
zipp                      3.7.0                    pypi_0    pypi
zlib                      1.2.11            h36c2ea0_1013    conda-forge
zstd                      1.5.2                ha95c52a_0    conda-forge
```

### Additional Context

I am running ubuntu 16.04 under WSL2

I currently cannot access packages from PyPi, https://files.pythonhosted.org is currently unaccesible to me due to an issue with the ISP. Unsure if this can be relevant in any way.

**Comment by ghylander** [comment]
Aside from this issue, I'm also having trouble installing conda-libmamba-solver, when I run the command through mamba I get:

`package conda-libmamba-solver-22.3.0-pyhd3eb1b0_0 requires libmambapy >=0.22.1, but none of the providers can be installed `

And when I run it through conda I get:

```
UnsatisfiableError: The following specifications were found to be incompatible with a past
explicit spec that is not an explicit spec in this operation (conda):

  - conda-libmamba-solver -> conda[version='>=4.12']
  - conda-libmamba-solver -> python[version='>=3.6'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.0.2l,<1.0.3a']

The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions

Package ca-certificates conflicts for:
python=3.9 -> openssl[version='>=1.1.1l,<1.1.2a'] -> ca-certificates
ca-certificates
certifi -> python -> ca-certificates
mamba -> openssl[version='>=1.1.1k,<1.1.2a'] -> ca-certificates
openssl -> ca-certificates
pip -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates

Package bzip2 conflicts for:
conda[version='>=4.12.0'] -> python[version='>=3.10,<3.11.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']
mamba -> libarchive[version='>=3.5.1,<3.6.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']
conda-libmamba-solver -> python[version='>=3.6'] -> bzip2[version='>=1.0.8,<2.0a0']
certifi -> python[version='>=3.10,<3.11.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']
pip -> python[version='>=3.10,<3.11.0a0'] -> bzip2[version='>=1.0.8,<2.0a0']

Package _libgcc_mutex conflicts for:
openssl -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main']
python=3.9 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main']
mamba -> libgcc-ng[version='>=9.4.0'] -> _libgcc_mutex==0.1=conda_forge

Package libsolv conflicts for:
mamba -> libsolv[version='>=0.7.19,<0.8.0a0']
conda-libmamba-solver -> libmambapy[version='>=0.22.1'] -> libsolv[version='>=0.7.20,<0.8.0a0']

Package futures conflicts for:
mamba -> conda[version='>=4.8'] -> futures[version='>=3.0.0']
conda[version='>=4.12.0'] -> conda-package-handling[version='>=1.3.0'] -> futures

Package setuptools conflicts for:
pip -> setuptools
conda-libmamba-solver -> conda[version='>=4.12'] -> setuptools[version='>=31.0.1']
mamba -> conda[version='>=4.8'] -> setuptools[version='>=31.0.1']
conda[version='>=4.12.0'] -> setuptools[version='>=31.0.1']
python=3.9 -> pip -> setuptools

Package libuuid conflicts for:
pip -> python[version='>=3.10,<3.11.0a0'] -> libuuid[version='>=1.0.3,<2.0a0']
conda[version='>=4.12.0'] -> python[version='>=3.10,<3.11.0a0'] -> libuuid[version='>=1.0.3,<2.0a0']
conda-libmamba-solver -> python[version='>=3.6'] -> libuuid[version='>=1.0.3,<2.0a0']
certifi -> python[version='>=3.10,<3.11.0a0'] -> libuuid[version='>=1.0.3,<2.0a0']

Package wheel conflicts for:
pip -> wheel
python=3.9 -> pip -> wheel

Package certifi conflicts for:
pip -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']
certifi

Package conda conflicts for:
mamba -> conda[version='>=4.8']
conda[version='>=4.12.0']

Package openssl conflicts for:
mamba -> libarchive[version='>=3.5.1,<3.6.0a0'] -> openssl[version='>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1l,<1.1.2a']
pip -> python[version='>=3.10,<3.11.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.0.2l,<1.0.3a|>=1.1.1b,<1.1.2a']
mamba -> openssl[version='>=1.1.1k,<1.1.2a']
python=3.9 -> openssl[version='>=1.1.1h,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a']
certifi -> python[version='>=3.7,<3.8.0a0'] -> openssl[version='1.0.*|>=1.0.2m,<1.0.3a|>=1.0.2n,<1.0.3a|>=1.0.2o,<1.0.3a|>=1.0.2p,<1.0.3a|>=1.1.1a,<1.1.2a|>=1.1.1b,<1.1.2a|>=1.1.1c,<1.1.2a|>=1.1.1d,<1.1.2a|>=1.1.1e,<1.1.2a|>=1.1.1g,<1.1.2a|>=1.1.1j,<1.1.2a|>=1.1.1k,<1.1.2a|>=1.1.1l,<1.1.2a|>=1.1.1i,<1.1.2a|>=1.1.1h,<1.1.2a|>=1.1.1m,<1.1.2a|>=1.0.2l,<1.0.3a']
opensslThe following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.27=0
  - feature:|@/linux-64::__glibc==2.27=0
  - openssl -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - python=3.9 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']

Your installed version is: 2.27
```

Could not find anything relevant when googling it, and "standard" fixes for `package X requires Y but none of the providers can be installed` did not work. Is this a known issue?

**Comment by jezdez** [comment]
Hi @ghylander, could you please file a separate issue for the conda-libmamba-solver installation comment you left above using the "Libmamba Solver Feedback (Experimental Feature)" issue template on https://github.com/conda/conda/issues/new/choose?

Regarding the original topic of this ticket, please provide more details which steps you took and if possible a traceback or error message so that we can look into the cause of your problem. Thank you!

**Comment by ghylander** [comment]
HI,
I continued trying ways to fix it, and finally solved it by simply running `conda update --all`. I'm not sure what changed, but now calling pip works.

Anyways, the behaviour was a bit strange. When I tried to install any package via pip command (`using python3.10 -m pip `instead worked) command, I would get a "bad interpreter" error, pointing towards `~/miniconda/env/envname/bin/python3.1`

An interpreter named "python3.1" existed (and still exists), which is simply an alias to the python3.10 interpreter. Running python3, python3.1 and python3.10 all started the python 3.10 interpreter.

Anyways, as I said, I run conda update --all simply to update my already installed packages and it solved the issue with using pip
